consensus/src/simplex/signing_scheme/bls12381_multisig.rs

```diff
@@ -23,57 +23,72 @@ use commonware_cryptography::{
         },
         variant::Variant,
     },
-    Digest,
+    Digest, PublicKey,
 };
-use commonware_utils::quorum_from_slice;
+use commonware_utils::set::Ordered;
 use rand::{CryptoRng, Rng};
 use std::{collections::BTreeSet, fmt::Debug};
 
 /// BLS12-381 multi-signature implementation of the [`Scheme`] trait.
 #[derive(Clone, Debug)]
-pub struct Scheme<V: Variant> {
-    /// Participant set used for signer indexing and batch verification.
-    participants: Vec<V::Public>,
-    /// Optional local signing key paired with its participant index.
+pub struct Scheme<P: PublicKey, V: Variant> {
```

Comment:
I've reduced everything I don't love about this PR to a suboptimal `Ordered` abstraction. Playing with some ideas there to get this over the finish line.